docs/guide.md

```diff
@@ -87,6 +87,34 @@ where `<API_KEY>` refers to the full generated API key (see [Creating and managi
 
 To know under which conditions access is granted, please see [Grant scheme](security.md#grant-scheme).
 
+#### Custom keyword
+
+ If you want to use a different keyword in the header, such as `Bearer`, simply subclass `KeyParser` set the `keyword` class variable. Next subclass `BaseHasAPIKey` set the `key_parser` class variable to use your new subclass.
```

Comment:
Hmm. It's a weird flow bc we're mentioning and explaining how to use key parser customization much later in the page, here: https://florimondmanca.github.io/djangorestframework-api-key/guide/#api-key-parsing

I think we can get away with this in this PR, and I can come up with some cleanup wording to review afterwards.

Reply:
I think i was able to update the documentation in a way you are suggesting. let me know what you think.